lib/haml/temple_engine.rb

```diff
@@ -0,0 +1,113 @@
+require 'temple'
+
+module Haml
+  class TempleEngine < Temple::Engine
+    define_options(
+      :attr_wrapper         => "'",
```

Comment:
Fixed in https://github.com/k0kubun/haml/commit/57c6b43b2a18493dd1e0e93086d6e5b18f375d7b